presto-main/src/test/java/com/facebook/presto/operator/BenchmarkHashBuildAndJoinOperators.java

```diff
@@ -257,8 +257,8 @@ public LookupSourceFactory benchmarkBuildHash(BuildContext buildContext)
         }
         operator.finish();
 
-        if (!operator.isFinished()) {
-            throw new AssertionError("Expected hash build operator to be finished");
+        if (!hashBuilderOperatorFactory.getLookupSourceFactory().createLookupSource().isDone()) {
```

Comment:
Benchmark was broken before. `LookupSource` is cached in `hashBuilderOperatorFactory`, so it's not recreated. Anyway, we need to benchmark hash building and part of it is `LookupSource` creation